src/components/Restaurant/Restaurant.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import './Restaurant.css'
+import Rating from '../Rating/Rating'
+
+class Restaurant extends Component {
+    render() {
+        const rest = this.props.rest;
+
+        return (
+            <tr className='rest'>
+                <td className='cuisine'>{ rest.cuisine }</td>
+                <td>
+                    <>
+                        <td className='rest-name'>{rest.name}
+                            {rest.tenbis ? <img src={require('../../images/tenbis.jpg')} className='tenbis'/> : null}
```

Comment:
I personally like making these kinds of things with pure CSS.
For example, if `rest.tenbis === true`, set another `className` (for convenience, check out: https://github.com/JedWatson/classnames) on the `td`, and make it display something with CSS.

For example:
```css
.rest-name.rest-tenbis::after {
  content: url('../../images/tenbis.jpg');
}
```

See:
- https://developer.mozilla.org/en-US/docs/Web/CSS/::after